android/app/build.gradle

```diff
@@ -51,7 +51,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.frb_example"
+        applicationId "com.itchysats.10101"
```

Comment:
Reading the guideline, referenced in the TODO above it seems this application id does not adhere to the last two rules.

    It must have at least two segments (one or more dots).
    Each segment must start with a letter.
    All characters must be alphanumeric or an underscore [a-zA-Z0-9_].


https://developer.android.com/studio/build/configure-app-module#set-application-id

Reply:
it adheres to the last one - the only trouble is the `10101` string. 
I guess this is a question of branding - should we go consistently with `10101` or `TenTenOne` everywhere, or only fallback for the places where we can't use the numbers? @bonomat ?

I will leave this one open and create a subset of this for macos/ios for the demo purposes, as I don't think it breaks those guidelines (I will doublecheck first).

Reply:
It works well on iOS (the new on the right is after the change)

![image](https://user-images.githubusercontent.com/8319440/203462275-96534461-94e3-42a4-b5d2-e3637140ff7b.png)